docs/resources/aviatrix_controller_cert_domain_config.md

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "Settings"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_controller_cert_domain_config"
+description: |-
+  Creates and manages an Aviatrix controller cert domain config
+---
+
+# aviatrix_controller_cert_domain_config
+
+The **aviatrix_controller_cert_domain_config** resource allows management of an Aviatrix Controller's cert domain config. This resource is available as of provider version R2.19+.
+
+## Example Usage
+
+```hcl
+# Create an an Aviatrix controller cert domain config
+resource "aviatrix_controller_cert_domain_config" "test" {
+  cert_domain = "abc.com"
+}
+```
+
+
+## Argument Reference
+
+The following argument is supported:
+
+* `cert_domain` - (Optional) Domain name that is used in FQDN for generating cert. Default value: "aviatrixnetwork.com".
```

Comment:
Is it required?

Reply:
It's optional. I corrected it in resource_aviatrix_controller_cert_domain_config.go